Source/Urho3D/Graphics/Octree.cpp

```diff
@@ -247,7 +247,7 @@ void Octant::GetDrawablesInternal(OctreeQuery& query, bool inside) const
         query.TestDrawables(start, end, inside);
     }
 
-    for (auto child : children_)
+    for (const auto &child : children_)

```

Comment:
Use [Coding conventions](https://urho3d.github.io/documentation/HEAD/_coding_conventions.html)
> Pointers and references append the * or & symbol to the type without a space in between. For example Drawable* drawable, Serializer& dest

Reply:
> children_ is array of pointers. There is no optimization

SharedPtr copying is relatively expensive.
I would go either for `const auto&` or explicit `Octant*`

Reply:
Octant* children_[NUM_OCTANTS]{};

Reply:
Oh, these are plaint pointers. Nasty.
Yep, in this case there's no optimization.